doc/source/parameters/cfd/post_processing.rst

```diff
@@ -70,9 +74,23 @@ This subsection controls the post-processing other than the forces and torque on
 * ``calculate average velocities``: controls if calculation of time-averaged velocities is enabled.
     * ``initial time``: initial time used for the average velocities calculations.
 
-* ``calculate pressure drop``: controls if calculation of the pressure drop from the inlet boundary to the outlet boundary is enabled. 
+* ``calculate pressure drop``: controls if calculation of the pressure drop from the inlet boundary to the outlet boundary is enabled.
     * ``inlet boundary id`` and ``outlet boundary id``: define the IDs for inlet and outlet boundaries, respectively. 
     * ``pressure drop name``: output filename for pressure drop calculations.
+    * The pressure drop :math:`\Delta P` and total pressure drop :math:`\Delta P_\text{total}` are calculated as such, with :math:`\Gamma` representing the boundary, :math:`\pmb{u}` the velocity  and :math:`P` the pressure:
+
+.. math::
+    \Delta P =  \frac{ \int_{\Gamma_\text{inlet}} P d \Gamma}{\int_{\Gamma_\text{inlet}} 1 d \Gamma} - \frac{ \int_{\Gamma_\text{outlet}} P d \Gamma}{\int_{\Gamma_\text{outlet}} 1 d \Gamma}
+
+.. math::
+    \Delta P_\text{total} =  \frac{ \int_{\Gamma_\text{inlet}} (P + \pmb{u} \cdot \pmb{u}) d \Gamma}{\int_{\Gamma_\text{inlet}} d \Gamma} - \frac{ \int_{\Gamma_\text{outlet}} (P + \pmb{u} \cdot \pmb{u}) d \Gamma}{\int_{\Gamma_\text{outlet}} d \Gamma}
```

Comment:
If I am not mistaken, the pressure drop component coming from the velocity should be multiplied by 1/2. Also, should we take only the square of the velocity normal to the boundary (I am not sure about this though).

Reply:
yeah I also think it should be the normal velocity component but I am also not 100% sure aboutthat.

Reply:
I looked into it a bit, and it seems that since dynamic pressure relates to the kinetic energy of the fluid we should use the magnitude of the velocity.